src/main/java/api/clients/ApiBoardClient.java

```diff
@@ -51,4 +51,17 @@ public BoardResponse updateBoardName(String boardId, String boardName, int expec
                 .body()
                 .extract().as(BoardResponse.class);
     }
+
+    @Step("Open or close board with id: {boardId}. Expected status code: {expectedStatusCode}")
+    public BoardResponse doOpenOrCloseExistBoard(String boardId, boolean openOrClose, int expectedStatusCode) {
+        return given()
+                .spec(requestSpec)
+                .queryParam("closed", openOrClose)
+                .put("/1/boards/{id}", boardId)
+                .then()
+                .statusCode(expectedStatusCode)
+                .log()
+                .body()
+                .extract().as(BoardResponse.class);
+    }
```

Comment:
Try making a method that accepts Builder as a Body. This practice is currently used by @SoniaMiach